src/scenes/AcceleratorRouter/MatrixView/ColumnsModal.tsx

```diff
@@ -465,34 +470,64 @@ export default function ColumnsModal(props: ColumnsModalProps): JSX.Element {
               />
 
               <List dense>
-                {filteredVariables.map((variable) => (
-                  <ListItem
-                    key={variable.id}
-                    sx={{
-                      px: 0,
-                      py: 0.5,
-                      '&:hover': { backgroundColor: 'action.hover' },
-                      borderRadius: 1,
-                    }}
-                  >
-                    <FormControlLabel
-                      control={
-                        <Checkbox
-                          checked={selectedColumns.includes(variable.stableId)}
-                          onChange={handleVariableToggle}
-                          value={variable.stableId}
-                          size='small'
-                        />
-                      }
-                      label={
-                        <Box>
-                          <Typography variant='body2'>{variable.name}</Typography>
-                        </Box>
-                      }
-                      sx={{ width: '100%', m: 0 }}
-                    />
-                  </ListItem>
-                ))}
+                <>
+                  {baseColumns.map((col) => (
+                    <ListItem
+                      key={col}
+                      sx={{
+                        px: 0,
+                        py: 0.5,
+                        '&:hover': { backgroundColor: 'action.hover' },
+                        borderRadius: 1,
```

Comment:
Might be worth a separate component for this (even if in the same file), since this is the same as what's rendered for `filteredVariables`